app/assets/javascripts/modules/track_search.js

```diff
@@ -0,0 +1,24 @@
+(function(Modules) {
+  "use strict";
+
+  Modules.TrackSearch = function() {
+    var that = this;
+
+    that.start = function(container) {
+
+      var filterSearchInput = container.find('table'),
```

Comment:
A table is not a`filterSearchInput`.

The two var declarations and the event listener can be reduced to:
`container.on('keyup', '.js-filter-table-input', trackSearch);`